pkgs/development/python-modules/pygments/default.nix

```diff
@@ -1,7 +1,7 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
+  fetchFromGitHub,
```

Comment:
Ooh, I wouldn't switch to github just because you can. Because python devs build off of pypi themselves, sometimes you can get problems with compat with other packages in the ecosystem. Hence I'd recommend against it. I [caused a big breakage](https://github.com/NixOS/nixpkgs/issues/425335) when I did that...

Reply:
Well, in general, fetching the source from GitHub is recommended (even for python package) whenever possible.
Of course, we can do that in a separate PR.